### ISSUE TITLE
Your experience

### ISSUE THREAD

**Issue by michal-sawluk-clurgo** [dinvlad/HyperVision#2]
Hey I was looking for solution similar to [Govee's](https://eu.govee.com/products/govee-envisual-tv-backlight-t2?irclickid=VIc013zfXxyNUakRwUzHdQtAUkAx0F0ktX1Ux00&irgwc=1&utm_source=impact_12845&utm_medium=affiliate&utm_campaign=digidip%20GmbH) and I found your project which looks very promising. And here is my question did you have any recording of how it works which you could share with us? I was wondering how fast it is and how good color reproduction comparing to Govee's.

**Comment by dinvlad** [comment]
Hi @michal-sawluk-clurgo - I've covered my experiences with it in #1 - please take a look there.

Also, here're a couple recordings of it I made back then - I no longer use the camera, but I do use it with a static color:
https://youtu.be/tIwtWI8xl1A
https://youtu.be/qeuAu0Nng18

**Comment by michal-sawluk-clurgo** [comment]
Thank for your response and your uploads, the effect is stunning. What is on top of your tv? Looks very similiar to Gooves T2, have you put there your fisheye camera?

**Comment by dinvlad** [comment]
@michal-sawluk-clurgo that's just my webcam for video calls :-) The fisheye PiCam was temporarily installed underneath the TV - hence you can see some wires in the video there. I cannot say that it was vastly better than Govee tbh, and I've heard Govee has since released improved versions that are probably better now. But I was also considering just getting AmbiVision or better still a 4K capture card and splitter, for a possibly much better look and experience.

Tbh I went with camera solution only because 4k equipment was not available at that time. But now 4k cards and splitters are cheap, so it would provide better quality and probably be cheaper to use that instead now.

**Comment by michal-sawluk-clurgo** [comment]
Well for me capture card is not an option, since I use built in smart tv a lot. And honestly comparing your solution to Govees T2 I would say yours its better, and since I have raspberry pi already I can build it for half of the price. Thanks for your answers, very appreciate ;)

**Comment by dinvlad** [comment]
Sounds like it's a worthwhile project then! Just note that price can be a little deceptive, for me at least it turned out more expensive than anticipated. Best of luck!

**Comment by michal-sawluk-clurgo** [comment]
Thanks :), one more question have you considered any other cameras for better color reproduction, that's the last thing i miss.

**Comment by dinvlad** [comment]
Yeah, I've tried a bunch, and they all seem to be roughly the same. For me, the problem with color reproduction is not so much in the quality of the cameras themselves, but in the number of pixels that are read out for each particular portion of the screen. Because of the sharp angle fisheye camera "looks" at the screen (either from top or the bottom), the number of pixels is just too small to accurately estimate the color.

So in my setup, color reproduction is better at the bottom of the screen, for example, where the number of pixels is greater, and the worst at the upper corners of the screen, where there're only a few pixels. You can see the same problem with Govee or any other camera that's looking at the screen at a sharp angle. That's actually the main reason why I think this approach is bound to failure.. The only way to fix it would be to install the camera at a distance from the screen, where the angle is closer to 90°.

**Comment by michal-sawluk-clurgo** [comment]
Alright then Ill buy the one u mentioned in [#1](https://github.com/dinvlad/HyperVision/issues/1). Currently my living room is being renovated so I can place this camera whenever I want. Do you think opposite wall close to ceiling in 2,7m distance will result with better outcome?
